.travis.yml

```diff
@@ -4,8 +4,6 @@ services:
   - docker
 
 before_script:
-    - bash ./docker/CISetup.sh $token $subBotID1 $subBotID2 $subBotID3 $subBotID4 $subBotID5 $subBotID6 $subBotToken1 $subBotToken2 $subBotToken3 $subBotToken4 $subBotToken5 $subBotToken6
```

Comment:
Builds on travis aren't getting tokens because travis needs to set them here

Reply:
I changed that, now the docker compose file passes the DISCORD_API_TOKEN and SUBBOTS directly to the container and the new config system will overwrite the token and subBots config. My fix was to use a bash script to do set the two new vars, but they don't allow me to set global ones.

I can change the compose file, but it is easier and more logical to set them correct in travis.